corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -331,6 +331,41 @@ SingleSelectEntry.prototype.onPreProcess = function (newValue) {
     }
 };
 
+/**
+ * Represents the label part of a Combined Multiple Choice question in a Question List
+ */
+function ChoiceLabelEntry(question, options) {
+    var self = this;
+    EntrySingleAnswer.call(this, question, options);
+    self.choices = question.choices;
+    self.templateType = 'choice-label';
+
+    self.hideLabel = ko.observable(options.hideLabel);
+
+    self.colStyle = ko.computed(function () {
+        var colWidth = parseInt(12 / self.choices().length) || 1;
```

Comment:
What is the significance of this 12 number?

Reply:
we use a 12 column grid system in our layout...